interface-elements-for-web/articles/rich-text-editor/table-of-contents.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Table of Contents
+
+---
+# Table of Contents
+
+Table of Contents allow navigate through a document's marked content (paragraphs, images, tables, and equations). Click a table's row while pressing CTRL to move to the heading or figure.
```

Comment:
allow navigate? Allow who or what to navigate?

Reply:
changed to 'The Table of Contents allows navigating through'

Reply:
The Table of Contents allows you to navigate through